packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -156,12 +162,8 @@ async function applyCategoryTemplate(category, template_lines, month, force) {
           num_months = differenceInCalendarMonths(target_month, current_month);
         }
         if (num_months < 0) {
-          console.log(
-            `${category.name}: ${`${template.month} is in the past:`} ${
-              template.line
-            }`,
-          );
-          return null;
+          errors.push(`${category.name}: ${`${template.month} is in the past.`}`);
```

Comment:
Can you remove the `${category.name}:` from all of the error messages and then add them back in after calling `applyCategoryTemplate`? That way there isn’t as much duplication.